homework/src/index.js

```diff
@@ -30,18 +33,84 @@
+  const repoSelect = createAndAppend("select", divSelect);
+  const divFlex = createAndAppend("section", root);
+  const divInfo = createAndAppend("article", divFlex, { id: "divInfo" });
+  const divCont = createAndAppend("article", divFlex, { id: "divCont" });
+
+  function main(HYF_REPOS_URL) {
+
+    fetchJSON(HYF_REPOS_URL)
+      .catch(error => {
+        const divError = createAndAppend("container", root, { id: "divError" });
+        divError.innerHTML = "";
+        divError.innerHTML = error;
+        return;
+      })
+      .then(repositories => {
+        repositories.forEach((repository, index) => {
+          createAndAppend("option", repoSelect, { html: repository.name, value: index });
+        });
+        repoSelect.addEventListener("change", (event) => {
+          const optionIndex = event.target.value;
+          renderRepository(divInfo, repositories[optionIndex]);
+          renderContributors(divCont, repositories[optionIndex]);
+        });
+        const initialRender = repositories[0];
+        renderRepository(divInfo, initialRender);
+        renderContributors(divCont, initialRender);
       }
-    });
+      );
   }
 
-  const HYF_REPOS_URL = 'https://api.github.com/orgs/HackYourFuture/repos?per_page=100';
+  function renderRepository(divInfo, repository) {
+    divInfo.innerHTML = "";
+    const table = createAndAppend("table", divInfo);
+    const tBody = createAndAppend("tBody", table);
+
+    const trRepository = createAndAppend("tr", tBody);
+    createAndAppend("td", trRepository, { html: "Repository : " });
+    const tdRepoLink = createAndAppend("td", trRepository, );
+    const linkRepository = createAndAppend('a', tdRepoLink, { html: repository.name, href: repository.html_url });
+
+    const trDescription = createAndAppend("tr", tBody);
+    createAndAppend("td", trDescription, { html: "Description : " });
+    createAndAppend("td", trDescription, { html: repository.description });
 
+    const trForks = createAndAppend("tr", tBody);
+    createAndAppend("td", trForks, { html: "Forks : " });
+    const tdForksLink = createAndAppend("td", trForks, );
+    const linkForks = createAndAppend('a', tdForksLink, { html: repository.forks, href: repository.forks_url });
+
+    const trUpdated = createAndAppend("tr", tBody);
+    createAndAppend("td", trUpdated, { html: "Updated : " });
+    createAndAppend("td", trUpdated, { html: repository.updated_at.toLocaleString() });
+  }
+
+  function renderContributors(divCont, repository) {
+    divCont.innerHTML = "";
+    createAndAppend('h2', divCont, { html: "Contributions", class: "h2" });
+
+    fetchJSON(repository.contributors_url)
+      .catch(error => {
+        createAndAppend('div', root, { html: error.message, class: 'alert-error' });
+        return;
+      })
+      .then(contributors => {
+        contributors.forEach(contributor => {
+          const ulContributor = createAndAppend('ul', divCont);
```

Comment:
There should only be one (unordered) list (`ul`) for the entire list of contributors. Here you are creating a separate list for each contributor.